pyproject.toml

```diff
@@ -209,6 +209,7 @@ norecursedirs = [
     "docs[\\/]generated",
     "astropy[\\/]extern",
     "astropy[\\/]_dev",
+    "astropy[\\/]units[\\/]tests[\\/]data",
```

Comment:
This tells `pytest` not to import the new module, which avoids triggering the error during test collection.

Reply:
@mhvk, this was all that was needed to avoid errors during test collection, so the pull request is now ready for final review.

Reply:
Not quite sure why `pytest` should ever go into a `data` directory, but that can be for whenever we add another directory to it.